app/src/app/create-calculation/create-calculation.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+    <form [formGroup]="createForm" (ngSubmit)="create()" fxLayout="column">
+        <mat-form-field appearance="fill">
+            <mat-label>Name</mat-label>
+            <input matInput placeholder="Name" formControlName="name"><mat-error
```

Comment:
Please use newline on mat-error

Reply:
fixed sorry will never happen again D:

Reply:
I sure hope so, otherwise this might be your last pull request.